app/server/models/credit_transfer.py

```diff
@@ -296,7 +296,7 @@ def check_sender_transfer_limits(self):
         return relevant_transfer_limits
 
     def check_sender_has_sufficient_balance(self):
-        return self.sender_user and self.sender_transfer_account.balance - self.transfer_amount >= 0
```

Comment:
Why do these checks require a user _and_ sufficient balance?

Reply:
I was wondering the same thing so i removed lol!